package.json

```diff
@@ -12,8 +12,5 @@
   "dependencies": {
     "mineflayer": "^4.3.0",
     "sharp": "^0.29.3"
-  },
-  "devDependencies": {
-    "mineflayer-item-map-downloader": "file:./"
```

Comment:
This is used for self reference when importing with require. Without this doing `require('mineflayer-item-map-downloader')` in examples would not work. Removing this would remove this feature.

Reply:
I brought back this. I removed it because it prevents local install (`file:`) to install its dependencies for some reason